static/src/javascripts/projects/common/modules/ui/selection-sharing.js

```diff
@@ -22,26 +25,50 @@ define([
     mediator,
     template,
     sharingTemplate,
+    resultsTemplate,
+    socialIconsTemplate,
     svgs,
     debounce,
     throttle,
     some
 ) {
 
     var $body = bonzo(document.body),
+        wikiIcon = svgs('searchWiki', ['icon']),
```

Comment:
Do these need to be stored in variables? Looks like they are only used to populate the `selectionSharing` dict. It is probably clear enough to just put them straight into `selectionSharing`.